### ISSUE TITLE
Mage Cooldowns Display fades/dissapear while casting

### ISSUE THREAD

**Issue by Chazoure** [Hekili/hekili#4213]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

Mage - Frost

### Describe the Issue

To put it simply whenever i cast any spell in any mage spec the Cooldowns Display simply dissapears and reappears when the cast is finished, it's like it stops recommending CD's while casting.

Thanks in advance!

### How to Reproduce

I Enter the game on my mage and i start casting on a dummy and it happens

### Snapshot (Link)

https://pastebin.com/ZBddMJfZ

### Additional Information

Disabling all addons except Hekili does not help

**Comment by Hekili** [comment]
Thanks for the ticket.  This looks like a side-effect of recent changes to make the ancillary displays (Interrupts, Cooldowns, Defensives) more efficient.  I'll investigate.

**Comment by mLgz0rn** [comment]
Hi, just fyi.

This still seem to happen on my Arcane mage.
